extensions/kafka/src/main/java/io/deephaven/kafka/ingest/JsonNodeChunkAdapter.java

```diff
@@ -63,11 +63,10 @@ private static FieldCopier makeFieldCopier(
             case Double:
                 return new JsonNodeDoubleFieldCopier(fieldName);
             case Object:
-                if (dataType == String.class) {
-                    return new JsonNodeStringFieldCopier(fieldName);
-                } else {
-                    throw new UncheckedDeephavenException("Raw objects not supported for JSON");
-                }
+                return (dataType == String.class)
+                        ? new JsonNodeStringFieldCopier(fieldName)
+                        : new JsonNodeObjectFieldCopier(fieldName)
```

Comment:
It probably makes sense to check `dataType == com.fasterxml.jackson.databind.JsonNode`?

Reply:
Either that or a base class of that I guess (eg, Object).